backend/routes/bookings.js

```diff
@@ -32,16 +32,25 @@ router.get('/', async (req, res, next) => {
 
 router.post('/', async (req, res, next) => {
     try {
-        if (req.query.username && req.query.tourName) {
-            const updatedUser = await req.models.booking.insert_booking(req.query.username, req.query.tourName);
-            res.json(updatedUser);
+        if (req.body.username && req.body.tourName) {
+            const updatedUser = await req.models.booking.insert_booking(req.body.username, req.body.tourName);
+
+            if(updatedUser){
+                const check = await req.models.booking.getSpecificBooking(req.body.username, req.body.tourName);
+                res.status(201).json(check); //returns the content
+            }
+            else{
+                console.error('There was an error in POST /bookings');
+                res.status(400).json();
+            }
             next();
-        } else {
-            throw new Error('Make sure all needed data is encluded\n');
+        } else{
+            console.error('There was an error in POST /bookings');
+            res.status(400).json();
```

Comment:
Can you add the error message "Make sure all needed data is included" here in the json? It will help us debug and help frontend know what the problem is when they ping us